src/main/java/com/tabnine/binary/BinaryRun.java

```diff
@@ -28,7 +29,8 @@ public BinaryRun(BinaryVersionFetcher binaryFetcher) {
             List<String> metadata = new ArrayList<>(asList(
                     "--client-metadata",
                     "pluginVersion=" + Utils.getPluginVersion(),
-                    "clientIsUltimate=" + PlatformUtils.isIdeaUltimate()
+                    "clientIsUltimate=" + PlatformUtils.isIdeaUltimate(),
+                    "clientChannel=" + Config.CHANNEL
```

Comment:
This is high risk change. The binary won't run if he does not recognise that parameter